File: 백준/Bronze/1159. 농구 경기/농구 경기.swift
var characterDictionary = [Character : Int]()
(0..<Int(readLine()!)!).forEach { _ in
    let character = Array(readLine()!)[0]
    characterDictionary[character, default: 0] += 1
}

let result = characterDictionary.filter { $0.value >= 5 }
.map {$0.key }
.sorted(by: <)
.reduce("") { $0 + String($1) }

print(result.isEmpty ? "PREDAJA" : result)

File: 백준/Bronze/1159. 농구 경기/농구 경기.py
from collections import Counter

N = int(input())
character_list = [input()[0] for _ in range(N)]

counter = Counter(character_list)
filtered_list = list(filter(lambda x: x[1] >= 5, counter.items()))

result = sorted(map(lambda x: x[0], filtered_list))
print("".join(result) if len(result) != 0 else "PREDAJA")